### PR TITLE
[CNVS Upgrade] Replace instances of container-pod with pod

### PR DESCRIPTION
**Pull request by jfurrow**
This PR replaces all instances of `container-pod` classes with `pod` where necessary. I removed some class assignments that are no longer necessary.

### CODE REVIEW

File: src/js/components/PodDetail.js [hunk 1]
```diff
@@ -90,16 +90,13 @@ class PodDetail extends mixin(TabsMixin) {
     const {activeActionDialog} = this.state;
 
     return (
-      <div className="flex-container-col">
```
Comment (mlunoe) [inline src/js/components/PodDetail.js]: I think we might actually need this for the log view to grow to its parent

Comment (mlunoe) [inline src/js/components/PodDetail.js]: I think we might actually need this for the log view to grow to its parent

File: src/js/components/ServiceDetail.js [hunk 2]
```diff
@@ -117,17 +117,11 @@ class ServiceDetail extends mixin(InternalStorageMixin, TabsMixin) {
...
-          container-pod-divider-bottom-align-right
-          container-pod-short-top flush-bottom flush-top
-          service-detail-header media-object-spacing-wrapper
-          media-object-spacing-narrow">
-          <Breadcrumbs />
-          <ServiceInfo onActionsItemSelection={this.onActionsItemSelection}
-            service={service} tabs={this.tabs_getUnroutedTabs()} />
-          {this.tabs_getTabView()}
-        </div>
+      <div>
```
Comment (mlunoe) [inline src/js/components/ServiceDetail.js]: Same thing here

File: src/js/components/PodDetail.js (hunk 1 above)
Comment (jfurrow) [inline src/js/components/PodDetail.js]: The page flex properties need to be addressed in a separate PR; the `flex-container-col` class definition doesn't exist anymore.
